api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -302,11 +362,21 @@ private RecipientAddress validatedRecipient(final String contactEmail) throws Me
   }
 
   private void sendWithRetries(
-      String contactEmail, String subject, String description, String htmlMessage)
+      List<String> contactEmails, String subject, String description, String htmlMessage)
       throws MessagingException {
     final MandrillMessage msg =
         new MandrillMessage()
-            .to(Collections.singletonList(validatedRecipient(contactEmail)))
+            .to(
+                contactEmails.stream()
+                    .map(
+                        e -> {
+                          try {
+                            return validatedRecipient(e);
```

Comment:
since this is the only call to `validatedRecipient()`, we can change the exception type inside that method to ServerErrorException, making this try/catch/rethrow unnecessary

Reply:
I don't know the historical reason for other email sending method to catch a checkedE, and throw another checked Exception, so I leave that method as it, and just change the one I know. For my case, I don't need the upstream calling knowing there is a checked exception throwed.

Reply:
Make validatedRecipient not throw checked Exception.